README.md

```diff
@@ -171,5 +202,13 @@ The beginning of the table should look like
 
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT students.name, students.surname, day, lessons.name
+FROM timetable
+JOIN lessons
+ON timetable.lesson_id = lessons.id
+JOIN groups
+ON timetable.class_id = groups.student_id
```

Comment:
It should be `groups.class_id` instead of `groups.student_id`.

Reply:
yeah, it should. I changed it